src/date/expand-pattern.js

```diff
@@ -24,8 +24,30 @@ define([
  * - { datetime: "full" } returns "EEEE, MMMM d, y 'at' h:mm:ss a zzzz";
  * - { pattern: "dd/mm" } returns "dd/mm";
  */
+
 return function( pattern, cldr ) {
-	var result;
+	var result, getDateTime, skeleton, dateSkeleton = "", timeSkeleton = "", type;
```

Comment:
```diff
-       var result, getDateTime, skeleton, dateSkeleton = "", timeSkeleton = "",
+       var getDateTime, result, skeleton, type,
+               dateSkeleton = "",
+               timeSkeleton = "";
```

Note that variables that assign initial value are placed in their own lines. Note the alphabetical order.